internal/statemachine/common_test.go

```diff
@@ -993,6 +993,7 @@ func TestFailedMakeDisk(t *testing.T) {
 		err = stateMachine.makeDisk()
 		asserter.AssertErrContains(err, "Error writing disk image")
 		helperCopyBlob = helper.CopyBlob
+		os.Remove("pc.img")
```

Comment:
test_image_size_flag does similar things but doesn't have the explicit remove, would you comment on the difference?  Also consider defer.

Reply:
So this one is kind of silly. In line 992 here we set the OutputDir to "". This is done to increase the coverage by covering a different scenario of where to place the image. In addition to a defer I should make sure that the behavior is correct with no OutputDir set.